### PR TITLE
making easier to run via main and writing status output to stderr

### PR DESCRIPTION
**Pull request by jdnrg**
I want to be able to redirect the output, invoke like this.

```
python3.8 -m pyinsights.cli  -c ./examples/pyinsights_ssh_prod.yml  --profile prod -r us-east-1 > results.json
```

### CODE REVIEW

File: pyinsights/cli.py [hunk 1]
```diff
@@ -69,3 +69,6 @@ def run(cli_options: CliOptions) -> bool:
 def main() -> bool:
     args = parse_args()
     sys.exit(run(args))
+
+if __name__ == '__main__':
+    main()
```
Comment (homoluctus) [inline pyinsights/cli.py]: @jdnrg 

Thanks.
Oh... I forgot to write that lines.

File: pyinsights/helper.py [hunk 2]
```diff
@@ -129,5 +129,5 @@ def processing(msg: str, end: str = '') -> None:
     """
 
     processing_msg = f'{Accessory.Accent}{color()}{msg}{Accessory.End}{end}'
-    sys.stdout.write(processing_msg)
-    sys.stdout.flush()
+    sys.stderr.write(processing_msg)
+    sys.stderr.flush()
```
Comment (homoluctus) [inline pyinsights/helper.py]: @jdnrg 

This means that processing messages are treated as errors.
But this implicit not to write the message to stdout.
Is my guess right ?
If you don't want to write the message to stdout, I have a better idea this change.
My idea is to introduce a `-q, --quiet` option which specifies that no processing messages should be displayed.

Comment (jdnrg) [reply pyinsights/helper.py]: It is just a second channel, this allows for the json data to be redirected from stdout and the color output to remain on the stderr, which works great. 
I run now like this
```python3.8 -m pyinsights.cli -c ./examples/pyinsights_ssh_prod.yml --profile prod -r us-east-1 > results.json```

Comment (homoluctus) [reply pyinsights/helper.py]: Want to get only the resulting JSON data?

The processing message is not an error and should not be output to stderr.
In this case, It is better to introduce an option to suppress the processing of message output.

Comment (jdnrg) [reply pyinsights/helper.py]: https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr) 
`Standard error is another output stream typically used by programs to output error messages or diagnostics.`
this is diagnostics and stderr is perfect for it.

Comment (homoluctus) [reply pyinsights/helper.py]: Sorry.
I can't agree with your opinion, so close this pull request.

I want to implement the `--quiet` option to suppress the display of processing messages.

**Comment by homoluctus** [comment]
Added `--quiet` option to suppress progress messages.
Therefore, we can get only the query results.

**Pull request closed without merging**